### ISSUE TITLE
Issue with setTilt() and isRestrictedPanning

### ISSUE THREAD

**Issue by Nikituh** [CartoDB/mobile-sdk#206]
If the following two lines are are set when initializing a map view, anomalies appear:

```
map.options.isRestrictedPanning = true
map.setTilt(20.0f, 0.0f)
```

If either line is commented out, everything works as intended, including tilting.

Only appears to happen on Android devices. Cf. BuildingFloors sample in AdvancedMap.Kotlin

Images:

Initial view:
![screenshot_20180412-152504](https://user-images.githubusercontent.com/8685359/38677441-f71c4c66-3e66-11e8-86ed-08f168b55ed3.png)

Zoom in slightly. Expected result:
![screenshot_20180412-152609](https://user-images.githubusercontent.com/8685359/38677462-03fe49d4-3e67-11e8-847d-3d54e9816433.png)

Actual result:

![screenshot_20180412-152514](https://user-images.githubusercontent.com/8685359/38677488-1788775e-3e67-11e8-8e43-5a9a0be56b70.png)

**Comment by jaakla** [comment]
Anomalies means here that rotation-tilting with touch is ok, but touch-zoom jumps to very zoomed in view and does not allow any zooming anymore.
@mtehver

**Comment by Nikituh** [comment]
`map.options.isRestrictedPanning = false`

also fixes the other 0 zoom issue, where the map is in the bottom of the screen and initial zoom causes a "jump".

**Comment by mtehver** [comment]
Thanks. This issue is now fixed in the master branch and will be included in the next release (4.1.3?)
